pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -739,7 +739,7 @@ self: super: builtins.intersectAttrs super {
   }) super.haskell-language-server;
 
   # tests depend on a specific version of solc
-  hevm = dontCheck (doJailbreak super.hevm);
+  hevm = unmarkBroken (dontCheck (appendPatch ./patches/hevm-update-deps.patch super.hevm));
```

Comment:
2. We almost never use `unmarkBroken`.  Instead, you'll need to remove `hevm` from the broken list in `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml` and run `maintainers/scripts/haskell/regenerate-hackage-packages.sh`.
3. We try to never carry around `.patch` files in Nixpkgs because it bloats the repo.  If at all possible, we like to pull commits or PRs from upstream with `fetchpatch`.  There should be a bunch of examples of that in this file and `pkgs/development/haskell-modules/configuration-common.nix`.

Reply:
Fixed 2.